consensus/misc/eip1559/eip1559.go

```diff
@@ -141,7 +205,18 @@ func CalcBaseFee(config *params.ChainConfig, parent *types.Header, time uint64)
 	}
 	elasticity := config.ElasticityMultiplier()
 	denominator := config.BaseFeeChangeDenominator(time)
-	if config.IsHolocene(parent.Time) {
+	minBaseFeeLog2 := config.MinBaseFeeLog2()
```

Comment:
I don't think this needs to be part of the config because there's no case where we want to enforce this minimum on blocks created before this minimum is part of the block header. `denominator` and `elasticity` are handled this way because there's a portion of OP Stack history where EIP 1559 is in effect, but only with hardcoded values, not configuration passed via extra data.